machida3/machida.pony

```diff
@@ -772,14 +775,22 @@ primitive Machida
     let ps = @get_name(o)
     recover
       if not ps.is_null() then
-        let ret = String.copy_cstring(@py_bytes_or_unicode_as_char(ps))
+        let ps_size = @py_bytes_or_unicode_size(ps)
+        let ret = String.copy_cpointer(@py_bytes_or_unicode_as_char(ps), ps_size)
```

Comment:
Could this use the `py_bytes_or_unicode_to_pony_string` primitive method instead?

Reply:
Yes, I think this can be replaced.